chainercv/transforms/image/pca_lighting.py

```diff
@@ -0,0 +1,47 @@
+import numpy
+
+
+def pca_lighting(img, sigma, eigen_value=None, eigen_vector=None):
+    """Alter the intensities of input image using PCA.
+
```

Comment:
How about "Alter the intensities of color channels based on given principal components" ?
Perhaps, "AlexNet-style color augmentation" is enough because it it well known.